lib/extensions.js

```diff
@@ -231,6 +233,35 @@ module.exports = function(realmConstructor) {
         });
     }
 
+
+    // A default Realm instance
+    realmConstructor.default = function() {
+        var user;
+        var config;
+
+        if (realmConstructor.Sync) {
+            user = realmConstructor.Sync.User.current;
+            if (user === undefined) {
+                throw new Error("No user is logged in");
```

Comment:
Since this method returns a Promise I think it’s better to `return Promise.reject(new Error())` instead of throwing.

Reply:
And because `User.current` also throws it might be best to use `User.all` instead, or surround this with a try/catch that returns a rejected promise. Or just surround the body of the method in `new Promise` and that will take care of handling throw nerrors.